tools/depends/target/zlib/ZLIB-VERSION

```diff
@@ -1,5 +1,5 @@
 LIBNAME=zlib
-VERSION=1.2.12
-ARCHIVE=$(LIBNAME)-$(VERSION).tar.xz
-SHA512=12940e81e988f7661da52fa20bdc333314ae86a621fdb748804a20840b065a1d6d984430f2d41f3a057de0effc6ff9bcf42f9ee9510b88219085f59cbbd082bd
+VERSION=1.2.13
+ARCHIVE=$(LIBNAME)-$(VERSION).tar.gz
```

Comment:
stick to the xz, better compression. same with native

Reply:
Also for testing, you can use BASE_URL to provide an alternate to kodi mirrors.

```
BASE_URL=https://www.zlib.net
```

will use the url https://www.zlib.net/zlib-1.2.13.tar.xz